x-pack/plugins/transform/public/app/sections/transform_management/transform_management_section.tsx

```diff
@@ -155,6 +163,48 @@ export const TransformManagement: FC = () => {
             )}
             {typeof errorMessage === 'undefined' && (
               <AlertRulesManageContext.Provider value={getAlertRuleManageContext()}>
+                {transformIdsWithoutConfig ? (
+                  <>
+                    {deleteAction.isModalVisible && (
+                      <DeleteActionModal {...deleteAction} hasNoConfig={true} />
+                    )}
+
+                    <EuiCallOut color="warning">
+                      <p>
+                        <FormattedMessage
+                          id="xpack.transform.danglingTasksError"
+                          defaultMessage="Found {count} {count, plural, one {transform} other {transforms}} [{transformIds}] with no corresponding  {count, plural, one {configuration} other {configurations}}."
```

Comment:
Might be worth adding extra clarification in here, such as 

`Found 1 transform running for which no configuration exists. This transform cannot be recovered and should be deleted.`

@lcawl any thoughts?

Reply:
Pete's suggestion mentioned that it's "running", which made me wonder how it could be running without config info. I've therefore omitted that word from this suggestion:
```suggestion
                          defaultMessage="{count} {count, plural, one {transform is} other {transforms are}} missing configuration details: [{transformIds}] {count, plural, one {It} other {They}} cannot be recovered and should be deleted."
```
If it *is* somehow running and we need to make that clear, lmk!

Reply:
Updated here https://github.com/elastic/kibana/pull/123407/commits/3c0a8c0048f129cb3e1cbdba81fc1b54616e6685